numpy/lib/nanfunctions.py

```diff
@@ -763,20 +763,17 @@ def nanmedian(a, axis=None, out=None, overwrite_input=False, keepdims=False):
         the same shape and buffer length as the expected output, but the
         type (of the output) will be cast if necessary.
     overwrite_input : bool, optional
-       If True, then allow use of memory of input array (a) for
-       calculations. The input array will be modified by the call to
-       median. This will save memory when you do not need to preserve
-       the contents of the input array. Treat the input as undefined,
-       but it will probably be fully or partially sorted. Default is
-       False. Note that, if `overwrite_input` is True and the input
-       is not already an ndarray, an error will be raised.
+       If True, then allow use of memory of input array (a) for calculations.
```

Comment:
Docstrings and comments should wrap at 72 columns, not 79, [PEP 8 says so](https://www.python.org/dev/peps/pep-0008/#maximum-line-length).